web/src/input_pill.ts

```diff
@@ -268,7 +269,7 @@ export function create<T>(opts: InputPillCreateOptions<T>): InputPillContainer<T
                     !funcs.appendPill(pill),
             );
 
-            store.$input.text(drafts.join(", "));
+            store.$input.text(util.format_array_as_list(drafts, "long", "conjunction"));
```

Comment:
Do we have an example of this code path in action? It might make sense to use the short format here, since pills are small, or it might be the case that this is supposed to be a `,`, if it's meant to be parsed by software.

Reply:
I think it should be using ```.join(",")``` as it's meant to be parsed by software.

Reply:
OK let's do that then.

Reply:
Done.